paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -173,9 +173,33 @@ internal class FormViewModel @Inject internal constructor(
             }.flattenConcat(),
             hiddenIdentifiers,
             showingMandate,
-            userRequestedReuse
+            userRequestedReuse,
+            defaultValuesToInclude,
         ).filterFlow()
 
+    private val defaultValuesToInclude get(): Map<IdentifierSpec, String> {
```

Comment:
Might be worth to extract this method and test it in isolation. Your call.

Reply:
Done